.travis.yml

```diff
@@ -36,6 +36,10 @@ cache:
                 - netmap
                 - $HOME/doxygen-install
 
+compiler:
+        - gcc
+        - clang
+
```

Comment:
"travis: run all test with gcc first" patch looks like is not needed. I do not see in result matrix that all gcc tests run prior clang.

Reply:
Usually this change works like it should (all GCC first, then all clang), but on sometimes travis runs gcc/clang mixed. Also all travis examples files have "compiler" defined first and "env" after that.

Reply:
@psavol I also see gcc/clang intermixed. Let's split this patch out, so that we can check if it works or not later.

Reply:
I'm just running a test on v3 on private branch, and this works again OK. Anyway, I moved the patch the last in the set.